std_msgs/String caffe_solver_path
---
std_msgs/Int64 status